src/status_im/contexts/profile/edit/name/events.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.contexts.profile.edit.name.events
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as re-frame]))
+
+(re-frame/reg-event-fx :profile/edit-name
+ (fn [{:keys [db]} [name]]
+   {:db            (assoc-in db [:profile/profile :display-name] name)
+    :json-rpc/call [{:method     "wakuext_setDisplayName"
```

Comment:
it would be nice to start adding integration tests for these RPC end points.

Reply:
@J-Son89 do we have any example test for RPC in project? i though they will add tests for endpoints in status-go.

Reply:
This pr is probably a good starting place to check as it should be clear of what other integration tests are doing.
https://github.com/status-im/status-mobile/pull/18304

While status go will do tests of their own it's best we add our own tests to cover as this ensures the mobile use cases are covered - 
it can be hard (and unnecessary) for status go to check certain tests cases and so by us "locking up" certain functionality we can help status go avoid making changes that will break for mobile.
Cc @ilmotta, @cammellos if there's any specific docs/examples you can direct @mohsen-ghafouri to?

Reply:
According to other example in the project I added an integration test for this event, could you please check again @J-Son89 ?

Reply:
Sure I'll take a look. Perhaps @ilmotta or @cammellos could take a look too as they probably have more real insight, I'm just a promoter :)

Reply:
@mohsen-ghafouri where did you add it? is that  events_test.cljs?

Reply:
@mohsen-ghafouri thank you, I think what @J-Son89 (correct me if I am wrong) was referring to, is the tests under   src/status_im/integration_test, which will actually call status-go

Reply:
Yeah, exactly. Perhaps we need some clearer documentation on these?

Reply:
I added profile_test under integration test.